injector/LogInjectorDesign.py

```
@@ -0,0 +1,354 @@
+                try:
+                    val = json.loads(node.value.value)
+
+                    if val["type"] == "adli_tag":
+                        ltInfo = self.ltMap[val["lt"]]
+                        if val["dir"] == "prev":
+                            newLineno = node.lineno - 1
+                        elif val["dir"] == "next":    
+                            newLineno = node.lineno + 1
+
+                        ltInfo["injectedLineno"] = newLineno
+                except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)